README.md

```diff
@@ -10,10 +10,10 @@ A central platform for hosting Ubuntu's various documentation portals.
 
 First [install Docker](https://www.docker.com/community-edition#/download). Linux users may need to add their user to the `docker` group.
 
-Then install [git-repo](https://github.com/esrlabs/git-repo) and [documentation-builder](https://github.com/canonicalltd/documentation-builder). E.g.:
+Then install [documentation-builder](https://github.com/canonicalltd/documentation-builder). E.g.:
 
-`` bash
-sudo snap install git-repo documentation-builder
+``` bash
+sudo snap install documentation-builder
```

Comment:
Actually I think this line is erroneous. `build-docs.sh` is run through `package.json`, so it's running inside the `canonicalwebteam/dev` image. And it gets there because `documentation-builder` is in `requirements.txt`.

So we should remove this step from the readme.